core/src/main/java/org/frankframework/http/HttpSessionBase.java

```diff
@@ -362,7 +364,7 @@ public void configureConnectionManager() {
 		connectionManager.setDefaultMaxPerRoute(getMaxConnections());
 
 		if (isStaleChecking()) {
-			log.info("set up connectionManager, setting stale checking ["+isStaleChecking()+"]");
+			log.info("set up connectionManager, setting stale checking [true]");
```

Comment:
Why not actually move that log line out of the if-condition? We don't log this message now if stale-checking is false. That's odd.

Reply:
Yea, saw it in JJ's PR, thought it was at least better this way, but I agree it's a bit silly...